forest/shared/src/cli/snapshot_fetch.rs

```diff
@@ -27,24 +27,50 @@ use url::Url;
 
 use crate::cli::to_size_string;
 
+pub enum FetchFlags {
+    NoCompression,
+    ChecksumValidation,
+    ForestServer,
+}
+
+impl From<FetchFlags> for bool {
+    fn from(flag: FetchFlags) -> bool {
+        match flag {
+            FetchFlags::NoCompression
+            | FetchFlags::ChecksumValidation
+            | FetchFlags::ForestServer => false,
+        }
+    }
+}
```

Comment:
This is not what I had in mind. I think we need to step back and re-think the overall approach.

Reply:
done